tests/test_sympymodule.py

```diff
@@ -67,3 +66,34 @@ def test_custom_function():
     mod = sympytorch.SymPyModule(expressions=[z], extra_funcs=extra_funcs)
     assert mod.sympy() == [z]
     assert mod(x=1, y=2) == 1 + 2 ** 2
+
+def test_rationals():
+    xvals = np.random.randn(100)
+    x = sympy.symbols('x')
+    y =  x*sympy.Rational(2,7)
+    mod = sympytorch.SymPyModule(expressions=[y])
+    assert mod.sympy() == [y], "mod: {}, y:{}".format(mod.sympy(), y)
+    assert len([p.item() for p in mod.parameters()])==0
+    y_tilde = mod(x=torch.tensor(xvals, dtype=torch.float64))[:,0]
+    error = y_tilde.detach().numpy()-xvals*2/7
+    assert (error**2).mean()<1e-10, "error:{}".format((error**2).mean())
```

Comment:
Nit: can this file be formatted as per PEP8? In particular including the appropriate whitespace around operators.
(If you're curious -- whilst I haven't set it up for sympytorch, on my more serious projects then this kind of thing is automatically enforced using pre-commit hooks, e.g. [here](https://github.com/patrick-kidger/diffrax/blob/main/CONTRIBUTING.md))